.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,44 @@
-## Description
+**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be compeleted as written. Provide known gaps to testing that may raise the risk of merging to production.
 
 
-## Original issue(s)
-department-of-veterans-affairs/va.gov-team#0000
+## Summary
 
+- *(Summarize the changes that have been made to the platform)*
+- *(If bug, how to reproduce)*
+- *(What is the solution, why is this the solution)*
+- *(Which team do you work for, does your team own the maintainence of this component?)*
+- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*
+
+## Related issue(s)
+
+- *Link to ticket created in va.gov-team repo*
+- *Link to previous change of the code/bug (if applicable)*
+- *Link to epic if not included in ticket*
 
 ## Testing done
 
+- *Describe what the old behavior was prior to the change*
+- *Describe the steps required to verify your changes are working as expected*
+- *Describe the tests completed and the results*
```

Comment:
I'm a little unclear on this, is this a reference to updating automated/unit tests and the behaviors that they are testing? Or is this about further integration testing?

Reply:
This is asking, what tests have you completed as part of the current change you are making. Does that make sense? If not, happy to discuss more.